gourmet/ImageExtras.py

```diff
@@ -32,11 +32,10 @@ def resize_image (image, width=None, height=None):
     else:
         return image
 
-def get_image_from_string (raw):
-    """Given raw image data, return an Image object."""
+def get_image_from_string(raw: bytes) -> Image.Image:
+    """Given raw image data (bytes), return an Image object."""
     if os.name =='posix':
-        sfi=io.StringIO()
-        sfi.write(raw)
+        sfi = io.BytesIO(raw)
         sfi.seek(0)
     else:
```

Comment:
This is tangential to the PR: Surely by now `BytesIO` also work non-posix, Windows namely, systems. `write_image_tempfile` could be removed?

Reply:
I think you're right - the [docs for Image.save()](https://pillow.readthedocs.io/en/latest/reference/Image.html#PIL.Image.Image.save) don't mention platform restrictions. But I wouldn't try to change Windows-specific branches until there's some Windows CI set up (Azure Pipelines & Appveyor can both run on Windows). And I wouldn't try to do that before the tests are passing on Linux. :slightly_smiling_face:

Reply:
Looks like Github actions can do Windows testing too - I haven't used that.

Reply:
Agreed. I plan on adding Windows CI at a later stage, when the application will be more stable.